src/multimeasurerest.ts

```diff
@@ -250,7 +250,7 @@ export class MultiMeasureRest extends Element {
       timeSig.point = this.render_options.number_glyph_point;
       timeSig.setTimeSig(timeSpec);
       timeSig.setStave(stave);
-      timeSig.setX(left + (right - left) * 0.5 - timeSig.getWidth() * 0.5);
+      timeSig.setX(left + (right - left) * 0.5 - timeSig.getTimeSig().glyph.getMetrics().width * 0.5);
```

Comment:
I admit to knowing nothing about this section of the code.

What was the bug?

How come the fixed line looks so different from the buggy line? From a cursory glance, it looks like the buggy line actually makes sense. :-) In fact, the fixed line looks weird to me, because why would we call getTimeSig() on timeSig??? :-)

Reply:
I thought the same. The original code was very similar to what we have now. I thought `getWidth` would return the same but obviously it does not :). The original was:
```
timeSig.timeSig.glyph.getMetrics().width
```

Reply:
The original code was using the width of just the glyph, the incorrect update used the entire width.

Suggestions below.